src/configs/gemm-config.c

```diff
@@ -268,6 +268,7 @@ static void init_pf32_gemm_config(void) {
     #if XNN_ENABLE_ARM_SME2
       const size_t mr = xnn_pf32_gemm_minmax_ukernel_32x32__neonsme2_get_mr();
       const size_t nr = xnn_pf32_gemm_minmax_ukernel_32x32__neonsme2_get_nr();
+      pf32_gemm_config.minmax.gemm[XNN_MR_TO_INDEX(1)] = xnn_init_hmp_gemm_ukernel((xnn_gemm_ukernel_fn) xnn_pf32_gemm_minmax_ukernel_1x32__neonsme2);
```

Comment:
For this.. the m_step x nstep_ of the GEMV kernel is 1x16VL.. So, it can be be scheduled as 1x256 which would be the ideal case for performance for this combination.  Of course, what is there in the PR would work as the packing formats are the same for the GEMM and GEMV. But something to think of for the next steps.